packages/react/src/misc/adapters/useProfilePriceController.ts

```diff
@@ -0,0 +1,23 @@
+import { UnspecifiedError } from '@lens-protocol/api-bindings';
+import { GetProfilePrice, ProfilePrices } from '@lens-protocol/domain/use-cases/profile';
+import { PromiseResult } from '@lens-protocol/shared-kernel';
+
+import { useSharedDependencies } from '../../shared';
+import { PromiseResultPresenter } from '../../transactions/adapters/PromiseResultPresenter';
+import { ProfilePriceGateway } from './ProfilePriceGateway';
+
+export type ProfilePriceResult = ProfilePrices;
```

Comment:
Do we need to alias it? The `Result` suffix could lead to confusion given the term is widely used in the context of `Result<T, E>`.

Reply:
`ProfilePrices` seems good to go out in the wild, it's DTO anyway.